plugins/techdocs-backend/src/search/DefaultTechDocsCollator.ts

```diff
@@ -108,6 +101,7 @@ export class DefaultTechDocsCollator implements DocumentCollator {
                 ...entityInfo,
                 path: doc.location,
               }),
+              path: doc.location,
```

Comment:
do we need path as well here when we have location just above?

Reply:
I think the problem is here that the location is only able to link to one of the two possible cases. If you search in context, you need to append the path to the current context (entity page or standalone route).

Reply:
Yeah, I think it's more likely that the `path` value will see even more usage in the future if we ever implement routeRef based routing on this.